tests/scr-support_scrolled-continuous/EPUB/package.opf

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<package xmlns="http://www.idpf.org/2007/opf" version="3.0" unique-identifier="pub-id" prefix="rendition: http://www.idpf.org/vocab/rendition/#">
+  <metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
+    <dc:coverage>Scripting</dc:coverage>
+    <dc:creator>Ivan Herman</dc:creator>
+    <dc:date>2022-06-02</dc:date>
+    <dc:description>Tests on handling overflow content with the "rendition:flow" property set to "scrolled-continuous". The publication is presented as one continuous scroll from spine item to spine item. This feature is optional if the reading system supports scripting.</dc:description>
```

Comment:
```suggestion
    <dc:description>Tests handling overflow content with the "rendition:flow" property set to "scrolled-continuous". The publication is presented as one continuous scroll from spine item to spine item. This feature is optional if the reading system supports scripting.</dc:description>
```

Reply:
I'm not 100% sure about this description. See my question about *why* this requirement exists elsewhere — my understanding is that the spec is saying that if scripting is supported, the scrolling behavior actually changes? If so I think the description should say that, and if not I think we're still missing some other subtlety. (It doesn't make sense to me that the test would only be testing scrolling *if* the test requires scripting *and* the rendition:flow is explicitly set — I think the point is that support for scripting causes some change in behavior?)

Probably this is because I don't understand the difference between scrolled-doc and scrolled-continuous. The package says scrolled-doc. I think if scripting is supported, it's supposed to behave as if it were scrolled-doc? (That's what your title suggests.) If so I think the description should say that explicitly.